packages/boxel-ui/addon/src/components/progress-bar/index.gts

```diff
@@ -30,26 +42,37 @@ export default class ProgressBar extends Component<Signature> {
   }
 
   <template>
-    <div
-      class='boxel-progress-bar'
-      data-test-boxel-progress-bar
-      aria-label={{@label}}
-      ...attributes
-    >
-      <div class='progress-bar'>
-        <div class='progress-bar-value' style={{this.progressWidth}}>
-          <div class='progress-bar-info {{this.progressBarPosition}}'>
-            <div class='progress-bar-label'>
-              {{#if @label}}
-                {{@label}}
-              {{/if}}
+    {{#if (eq @progressVariant 'circle')}}
```

Comment:
maybe have an outer class that has aria-label that is shared between the 2 variants